tenets/codelingo/code-review-comments/do-not-discard-errors/codelingo.yaml

```diff
@@ -0,0 +1,27 @@
+tenets:
+  - name: do-not-discard-errors
+    flows:
+      codelingo/docs:
+        title: Do Not Discard errors
+        body: |
+          Do not discard errors using _ variables. If a function returns an error, 
+          check it to make sure the function succeeded. Handle the error, return it, or, 
+          in truly exceptional situations, panic.
+      codelingo/review:
+        comment: |
+          Do not discard errors using _ variables
```

Comment:
Finish with a full stop. If ignoring the err is intentional, add a justification in a comment.